core_weekly/date_util.py

```diff
@@ -84,3 +90,15 @@ def compute_from_day_to_day_statement(self, date_range):
             first_day_part,
             last_day_part
         )
+
+    def does_year_begins_with_week_one(self, year):
+        """Some years begin with a week 1, others with a week 53.
+
+        :param year: Year
+        :type year: str
+        :rtype: bool
+        """
+        if year == 2021:
```

Comment:
Hardcoded list 😆  I don't know how to do it in a better way.

Reply:
I found something better ! https://github.com/PrestaShop/core-weekly-generator/pull/90/commits/f16bd44dbacce1383ca9caba73ae014f2b9b1fa9

I compute the 1st day of the year and verify the week number. For 2020, week number of 2020-01-01 is 1 while for 2021 week number of 2021-01-01 is 53

Reply:
Instead of using this method, we should be able to check the number of weeks per year instead.

Reply:
I'm not 100% sure this is reliable. Depending on if the 1st day is a Monday, Thursday or Sunday we might have years with 52 or 53 or 54 weeks, and it might not be linked to the 1st week number.

Reply:
We should follow the normal behaviour: 
![image](https://user-images.githubusercontent.com/1462701/126464318-258b9fa4-ffd7-4e72-89ea-75836dbd75ed.png)
The first week doesn't start at first day of January :thinking: